### PR TITLE
Update dependencies: protobuf, flac, zlib

### PR DESCRIPTION
**Pull request by banasraf**
Signed-off-by: Rafal <rbanas@nvidia.com>

### CODE REVIEW

**Review by jantonguirao — APPROVED** [review]

**Review by szalpal — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -142,8 +142,8 @@ The repository consists mostly of externally hosted subrepositories:
 .. _zlib: https://github.com/madler/zlib
 .. |zliblic| replace:: zlib License
 .. _zliblic: https://github.com/madler/zlib/blob/master/README
-.. |zlibver| replace:: 1.2.12
-.. _zlibver: https://github.com/madler/zlib/releases/tag/v1.2.12
+.. |zlibver| replace:: 1.2.13
+.. _zlibver: https://github.com/madler/zlib/releases/tag/v1.2.13
```
Comment (JanuszL) [inline README.rst:146]: Do we still need zlib-CVE-2022-37434.patch in such case?

**Comment by banasraf** [comment]
CUDA11 build: [6442577]

**Comment by banasraf** [comment]
CUDA10 build (passed): 6442582

File: README.rst (hunk 1 above)
Comment (banasraf) [reply README.rst:146]: Removed

**Comment by banasraf** [comment]
CUDA11 build (passed): [6455382]

**Pull request merged**